src/api/MainSplitter.cc

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2022, Antti Hyvarinen <antti.hyvarinen@gmail.com>
+ * Copyright (c) 2022, Seyedmasoud Asadzadeh <seyedmasoud.asadzadeh@usi.ch>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <LookaheadSplitter.h>
+#include "MainSplitter.h"
```

Comment:
```suggestion
#include "MainSplitter.h"
#include "LookaheadSplitter.h"
```
Header file corresponding to the `.cc` file should always be first.
And our headers we should include using `""`, just to be consistent.

Reply:
Sure